problemsets-mat223/modules/module4-exercises.tex

```diff
@@ -181,5 +181,67 @@
 				real numbers.
 			\end{enumerate}
 		\end{solution}
+	
+        	\prob Let $\vec u, \vec v \in \R^{n}$. In this problem, we will prove
+		$\Abs{\vec u \cdot \vec v}\leq \Norm{\vec u}\Norm{\vec v}$. This is called
+		the \emph{Cauchy-Schwarz inequality}.
+		\begin{enumerate}
+			\item Assuming the geometric definition
+				$\vec u \cdot \vec v = \Norm{\vec u}\Norm{\vec v}\cos \theta$
+				where $\theta$ is the angle between $\vec u$ and $\vec v$, prove
+				the \emph{Cauchy-Schwarz inequality}.
+
+			\item The \emph{Cauchy-Schwartz inequality} can also be proved using
+				\emph{only} the algebraic definition of the dot product. Keep in
+				mind the following facts which come from the algebraic definition:
+
+				\begin{enumerate}
+					\item Explain why the result is immediate if one (or both)
+						of $\vec u, \vec v$ is the zero vector.
+
+					\item Assume $\vec u, \vec v$ are non-zero vectors. Consider
+						the function $P: \R \to \R$ where $P(t) = \Norm
+						{t\vec u - \vec v}^{2}$. Convince yourself that
+						$P(t) \geq 0$ for all $t \in \R$.
+
+					\item Simplify $P(t)$ into a quadratic formula so that
+						$P(t) = at^{2}- bt + c$ by determining its coefficients
+						$a, b, c$ in terms of $\vec u, \vec v$.
+
+					\item Prove
+						$\Abs{\vec u \cdot \vec v}\leq \Norm{\vec u}\Norm
+						{\vec v}$, the \emph{Cauchy-Schwarz inequality}. Hint: Consider
+						$P(\frac{b}{2a})$.
+
+					\item Prove
+						$\Abs{\vec u \cdot \vec v}= \Norm{\vec u}\Norm{\vec v}$
+						if the vectors $\vec u, \vec v$ are scalar multiples of
+						each other.
+				\end{enumerate}
+		\end{enumerate}
+
+		\prob Let $\vec u, \vec v \in \R^{n}$. In this problem, we will prove
+		$\Norm{\vec u + \vec v}\leq \Norm{\vec u}+ \Norm{\vec v}$. This is called the
+		\emph{Triangle Inequality}.
+		\begin{enumerate}
+			\item Draw a picture and explain why it is called the \emph{Triangle Inequality}.
```

Comment:
-> Pick your favorite vectors `u` and `v` and draw a picture of `u`, `v`, and `u+v`. Root your vectors so that they form the edges of a triangle. Can you explain...